app/filters.js

```diff
@@ -0,0 +1,46 @@
+const dateFilter = require('nunjucks-date');
+
+module.exports = function (env) {
+  /**
+   * Instantiate object used to store the methods registered as a
+   * 'filter' (of the same name) within nunjucks. You can override
+   * gov.uk core filters by creating filter methods of the same name.
+   * @type {Object}
+   */
+  let filters = {}
+
+  filters.date = dateFilter;
+
+  function mojDateFilter(timestamp, type) {
+
+    // default datetime to timestamp
+    let datetime = timestamp;
+
+    switch(type) {
+      case "longdatetime":
```

Comment:
I'd suggest a terser default “datetime” and then prefix with “short” only for short date.